lib/jarbs.rb

```diff
@@ -151,6 +156,32 @@ def run
         end
       end
 
+      command :logging do |c|
+        c.syntax = 'jarbs logging NAME [enable|disable]'
+        c.summary = <<-DESC
+Enable CloudWatch to ElasticSearch logging. You will need to have a lambda available to push logs
```

Comment:
Yeah, we should, or at least have one available. This is meant to be OSS, so I can't hardcode it here. It's easy to provide though and once you provide it, it saves it for future use.